homeassistant/components/tradfri.py

```diff
@@ -110,15 +111,19 @@ def gateway_discovered(service, info):
 @asyncio.coroutine
 def _setup_gateway(hass, hass_config, host, key, allow_tradfri_groups):
     """Create a gateway."""
-    from pytradfri import cli_api_factory, Gateway, RequestError, retry_timeout
+    from pytradfri import Gateway, RequestError
+    from pytradfri.api.aiocoap_api import api_factory
```

Comment:
This will raise an import error when tinydtls is not installed correctly. If that happens, let's log it